src/m_menu.c

```diff
@@ -2608,7 +2608,7 @@ void M_DrawDefVerify()
   if (whichSkull) // blink the text
     {
       strcpy(menu_buffer,"Reset to defaults? (Y or N)");
-      M_DrawMenuString(VERIFYBOXXORG+8-WIDESCREENDELTA,VERIFYBOXYORG+8,CR_RED);
+      M_DrawMenuString(VERIFYBOXXORG+8-WIDESCREENDELTA,VERIFYBOXYORG+8,CR_NONE);
```

Comment:
Should actual warnings remain red?

Reply:
Yes, I think so. In the PWADs I've tested, red looks more right. Instructions too?